src/ngx_http_datadog_module.cpp

```diff
@@ -52,18 +52,13 @@ static char *merge_datadog_loc_conf(ngx_conf_t *, void *parent, void *child) noe
 
 using namespace datadog::nginx;
 
-#define DEFINE_DEPRECATED_COMMAND_DATADOG_TRACING(NAME, TYPE) \
-  {                                                           \
-    NAME, TYPE, warn_deprecated_command_datadog_tracing,      \
-        NGX_HTTP_LOC_CONF_OFFSET, 0, NULL                     \
-  }
-
 constexpr datadog::nginx::directive module_directives[] = {
-    DEFINE_DEPRECATED_COMMAND_DATADOG_TRACING("datadog_enable",
-                                              anywhere | NGX_CONF_NOARGS),
+    WARN_DEPRECATED_COMMAND("datadog_enable", anywhere | NGX_CONF_NOARGS,
```

Comment:
Shouldn't we move this to tracing/directives ?

Reply:
No, they are globally applied directives.